.ci/update-gh-pages.sh

```diff
@@ -7,18 +7,21 @@ if [ "$TRAVIS" != "true" ]; then
   return 1;
 fi
 
-if [ $TRAVIS_PULL_REQUEST != "false" ]; then
+if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
   # Dont build anything for PR requests, only for merges.
+  echo "Skip script since it processes a Pull Request, not a merge."
```

Comment:
These will all only be visible if `… > /dev/null 2>&1` is removed from `.travis.yml`, right?

Reply:
Yes

Reply:
Correct. I hope we do not need the logs since everything runs now as expected :wink: